libraries/CoreAudio/CoreAudio.cpp

```diff
@@ -150,7 +157,9 @@ void CoreAudio::action( int id ) {
         {
           if (soundPlayFlashRaw.isPlaying())
           {
-            soundPlayFlashRaw.stop();
+            // soundPlayFlashRaw.stop();
```

Comment:
please remove the commented line. It's fine to have the previous version in the history and a cleaner code